waspc/src/Wasp/Generator/FileDraft/WriteableMonad.hs

```diff
@@ -33,8 +36,17 @@ class (MonadIO m) => WriteableMonad m where
     FilePath ->
     m ()
 
+  copyDirectoryRecursive ::
```

Comment:
I know with copying dirs there is always a question about what happens with symlinks, esp. if they are pointing to something within a dir that is being copied. I guess this is not important for our case, right?

Reply:
This under the covers uses `Path.IO.copyDirRecur`, which we did before in Migrate. This does _not_ follow symbolic links. I think in this case, for now, it does not matter. I can add a comment on that though so they don't have to find the docs. 👍🏻

Reply:
Added the following comment to help clarify use:

```haskell
  -- | Copies a directory recursively.
  -- It does not follow symbolic links and preserves permissions when possible.
  copyDirectoryRecursive ::
```